src/core/featuremodel.h

```diff
@@ -31,6 +32,8 @@ class FeatureModel : public QAbstractListModel
 {
     Q_OBJECT
     Q_PROPERTY( QgsFeature feature READ feature WRITE setFeature NOTIFY featureChanged )
+    Q_PROPERTY( QgsFeature linkedParentFeature READ linkedParentFeature WRITE setLinkedParentFeature NOTIFY linkedParentFeatureChanged )
+    Q_PROPERTY( QgsRelation linkedRelation READ linkedRelation WRITE setLinkedRelation NOTIFY linkedRelationChanged )
```

Comment:
Do I understand this correctly, this is only set if this is a child feature and if it is set, the foreign key attributes will automatically be synchnronized with the attributes in the linkedParentFeature?

Reply:
yes

Reply:
Can you add a docstring/comment?